lib/ExpensiMark.js

```diff
@@ -162,6 +162,11 @@ export default class ExpensiMark {
                 },
                 replacement: g1 => `<blockquote>${g1}</blockquote>`,
             },
+            {
+                name: 'heading1',
+                regex: /^#(?!#) +(.+)$/gm,
+                replacement: '<h1>$1</h1>',
+            },
```

Comment:
This rule was misplaced. It should have been placed before quote rule, because the quote rule consumes the newlines chars i.e. `\n`. This made such markdown fail `> a\n# b`. (Coming from https://github.com/Expensify/App/issues/21846)